skyportal/handlers/api/spectrum.py

```diff
@@ -1,9 +1,7 @@
-import tornado.web
-from sqlalchemy.orm import joinedload
 from marshmallow.exceptions import ValidationError
 from baselayer.app.access import permissions, auth_or_token
 from ..base import BaseHandler
-from ...models import DBSession, Spectrum, Comment, Instrument, Source
+from ...models import DBSession, Spectrum, Instrument, Source, Candidate
```

Comment:
The changes to this file are due to the fact that the parent may now either be a comment or a candidate, so we add checks here